bundles/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/commandclass/ZWaveCommandClass.java

```diff
@@ -477,7 +478,8 @@ protected int extractValue(byte[] buffer, int offset, int size) {
         NON_INTEROPERABLE(0xF0, "NON_INTEROPERABLE", null),
 
         // MANUFACTURER_PROPRIETARY class definitions are defined by the manufacturer and device id
-        FIBARO_FGRM_222(0x010F, 0x0301, "FIBARO_FGRM_222", FibaroFGRM222CommandClass.class);
+        FIBARO_FGRM_222(0x010F, 0x0301, "FIBARO_FGRM_222", FibaroFGRM222CommandClass.class),
+        FIBARO_FGRM_222_1(0x010F, 0x0301, "FIBARO_FGRM_222", FibaroFGRM222CommandClass.class);
```

Comment:
Oops - the second one should be 302. I got in a muddle with having both OH1 and OH2 source all open in the IDE at the same time :(

Thanks.